build.gradle

```diff
@@ -71,7 +71,7 @@ cobertura {
 }
 
 test {
-    useJUnitPlatform()
+    useJUnit()
```

Comment:
In original repo, here is '''useJUnitPlatform()'''. But when I make some errors in a test case on purpose, then I run '''./gradlew test''', I still get '''BUILD SUCCESSFUL'''. Thus, I change it to '''useJUnit()''', I can get the error location after I run the command.

Reply:
Go back to Junit is not a good alternative.
https://www.baeldung.com/junit-5-gradle

We need to review the test dependency duplication. In build.Gradle 

https://github.com/amadeus4dev/amadeus-java/blob/4b5937996a340ffe2e446018dfe4de218d3ec62e/build.gradle#L36

Reply:
Let’s discuss tomorrow

Reply:
Great! So the idea is migrate the junit4 inside the repo to junit5? If so, it would be a good chance for me to study the new features on junit5

Reply:
> Go back to Junit is not a good alternative. https://www.baeldung.com/junit-5-gradle
> 
> We need to review the test dependency duplication. In build.Gradle
> 
> https://github.com/amadeus4dev/amadeus-java/blob/4b5937996a340ffe2e446018dfe4de218d3ec62e/build.gradle#L36

I just change the dependencies following this tutorial, and it works